spec/env.coffee

```diff
@@ -115,30 +115,31 @@ window.setupStore = (options) ->
   env = {}
 
   options = options or {}
-  container = env.container = new Ember.Container()
+  registry = new Ember.Registry(options)
```

Comment:
Could you please provide your Ember version? Well, all my tests are failed due to switching to `Ember.Registry` instead of using `Ember.Container` as first is `undefined` on `1.8.1`.